osf/utils/migrations.py

```diff
@@ -419,6 +419,7 @@ def state_forwards(self, app_label, state):
 
     def database_forwards(self, app_label, schema_editor, from_state, to_state):
         ensure_schemas(to_state.apps)
+        map_schemas_to_schemablocks(to_state.apps)
```

Comment:
`UpdateRegistrationSchemas` is used in [an earlier migration](https://github.com/CenterForOpenScience/osf.io/blob/1dde647bfca24b6034d35d0d5bc9baa78cf4e32c/osf/migrations/0146_update_registration_schemas.py#L43), which will break if `RegistrationSchemaBlock` doesn't exist in `to_state.apps`.

I'm not sure the best response to this -- maybe check whether `to_state.apps.get_model('osf.registrationschemablock')` raises `LookupError`? Or a new migration operation for the post-schema-block world?